examples/autoinstall.yaml

```diff
@@ -0,0 +1,16 @@
+version: 1
+early-commands:
+        - echo a
+        - sleep 1
+        - echo a
+late-commands:
+        - echo a
+        - sleep 1
+        - echo a
+keyboard:
+        layout: gb
+identity:
+        realname: ''
+        username: ubuntu
+        password: '$6$wdAcoXrU039hKYPd$508Qvbe7ObUnxoj15DRCkzC3qO7edjH0VV7BPNRDYK4QR8ofJaEEF2heacn0QgD.f8pO8SNp83XNdWG6tocBM1'
+        hostname: ubuntu
```

Comment:
Does this example include all possible keys?  We might want to expand this with comments or examples of other accepted autoinstall keys.

Reply:
No it certainly does not, and yes providing a more complete example would make sense.